src/main/java/com/khomishchak/cryptoportfolio/WalletSphereApplication.java

```diff
@@ -4,10 +4,10 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-public class CryptoPortfolioApplication {
+public class WalletSphereApplication {
```

Comment:
Do you keep in mind to have respective name of package and artifact id? Of course it's optional and only business-related(not sure that's even business stuff)